Rick-and-Morty/Rick And Morty/Views/CharactersView.swift

```diff
@@ -1,30 +1,28 @@
 import SwiftUI
 
 struct CharactersView: View {
-    let characters: [Character]
-    let title: String
+    let charactersViewState: CharactersViewState
     
     var body: some View {
         NavigationView {
             ScrollView(.vertical) {
                 VStack(alignment: .leading) {
-                    ForEach(characters, id: \.id) { character in
-                        CharacterCell(character: character, imagePosition: getImagePosition(character: character))
+                    ForEach(charactersViewState.characterViewStates) { characterViewState in
+                        CharacterCell(characterViewState: characterViewState)
                             .frame(minWidth: 0, maxWidth: .infinity, minHeight: 0, maxHeight: 120)
                     }
                 }
             }
-            .navigationBarTitle(title)
+            .navigationBarTitle(charactersViewState.title)
         }
     }
-    
-    func getImagePosition(character: Character) -> CharacterImagePosition {
-        character is Rick ? .right : .left
-    }
 }
 
+/*
 struct CharactersView_Previews: PreviewProvider {
     static var previews: some View {
-        CharactersView(characters: ricks, title: "Ricks")
+        //CharactersView(viewModel: CharactersViewModel(characterType: .rick))
+        //CharactersView(viewModel: CharactersViewModel(characterType: .morty))
     }
 }
+*/
```

Comment:
This should be removed. there is no need for commented out code. You have a repo to check for old versions if you really need to :)